addons/website/static/src/snippets/s_rating/options.js

```diff
@@ -124,17 +125,21 @@ options.registry.Rating = options.Class.extend({
      */
     _renderIcons: function () {
         const icons = {
-            'fa-star': 'fa-star-o',
-            'fa-thumbs-up': 'fa-thumbs-o-up',
-            'fa-circle': 'fa-circle-o',
-            'fa-square': 'fa-square-o',
-            'fa-heart': 'fa-heart-o'
+            'fa-star': 'fa-star',
+            'fa-thumbs-up': 'fa-thumbs-up',
+            'fa-circle': 'fa-circle',
+            'fa-square': 'fa-square',
+            'fa-heart': 'fa-heart'
```

Comment:
I don't know the point of this change but in any case, this is not good: creating an object whose keys are equal to its values... why?